CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -105,7 +105,7 @@ public class FlaCudaWriter : IAudioDest
 		TimeSpan _userProcessorTime;
 
 		// header bytes
-		// allocated by flake_encode_init and freed by flake_encode_close
+		// allocated by Flake_encode_init and freed by Flake_encode_close

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.